src/lib/djerba/plugins/supplement/body/test/WGTS.RESEARCH.supp.ini

```diff
@@ -0,0 +1,8 @@
+[core]
+author=PLACEHOLDER
+
+[supplement.body]
+assay=WGTS
+user_supplied_draft_date=2024/02/05
```

Comment:
Maybe validate the user supplied draft/signoff dates here: https://github.com/oicr-gsi/djerba/blob/9a8f86251cdf7b9df7536ca0f855585c8f17d478/src/lib/djerba/plugins/supplement/body/plugin.py#L49

Can use the `is_valid_date()` function here: https://github.com/oicr-gsi/djerba/blob/9a8f86251cdf7b9df7536ca0f855585c8f17d478/src/lib/djerba/util/date.py#L10

You would then need to change the dates in test data to eg. 2024-02-05 :smile: